conflowgen/analyses/yard_capacity_analysis.py

```diff
@@ -20,8 +21,8 @@ class YardCapacityAnalysis(AbstractAnalysis):
     def get_used_yard_capacity_over_time(
             self,
             storage_requirement: typing.Union[str, typing.Collection, StorageRequirement] = "all",
-            smoothen_peaks: bool = True
-    ) -> typing.Dict[datetime.datetime, float]:
+            smoothen_peaks: bool = True,
+    ) -> UsedYardCapacityOverTime:
```

Comment:
I checked the documentation at https://conflowgen--186.org.readthedocs.build/en/186/api.html#conflowgen.YardCapacityAnalysis.get_used_yard_capacity_over_time and saw that this return type is not clickable. Could you add it to the official API? An example of how it can look like you can find at https://conflowgen--186.org.readthedocs.build/en/186/api.html#conflowgen.ModalSplitAnalysis.get_modal_split_for_hinterland_traffic.